pkg/cortex/serve/cortex_internal/lib/storage/gcs.py

```diff
@@ -57,15 +57,33 @@ def is_valid_gcs_path(path: str) -> bool:
     def _does_blob_exist(self, prefix: str) -> bool:
         return isinstance(self.gcs.get_blob(blob_name=prefix), storage.blob.Blob)
 
+    def _gcs_matching_blobs_generator(self, max_results=None, prefix="", include_dir_objects=False):
+        for blob in self.gcs.list_blobs(max_results, prefix=prefix):
+            if include_dir_objects or (not include_dir_objects and not blob.name.endswith("/")):
```

Comment:
nit: I think this can be:

`if include_dir_objects or not blob.name.endswith("/"):`

Reply:
Yep.